packages/react-theme-sass/config/api-extractor.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
At least IMO, if this package is never intended to ship JS, the API Extractor configs and output should be removed. The unfortunate part is I'm not sure whether the migration generator understands this or will just keep trying to add them back.